Makefile

```diff
@@ -0,0 +1,31 @@
+run:
+	pipenv run python run.py
+
+install:
+	pipenv install --dev
+
+load_templates:
+	./scripts/load_templates.sh	
+
+docker-build:
+	docker build -t srm-rh-ui .
+
+docker_run:
+	docker run -p 9092:9092 srm-rh-ui
+
+docker_stop:
+	docker stop srm-rh-ui
+
+flake8:
+	pipenv run flake8 --exclude=whitelist.py 
+
+vulture:
+	pipenv run vulture .
+
+update_vulture_whitelist:
+	pipenv run vulture . --make-whitelist > whitelist.py
+
```

Comment:
Command works but I see this error
```
ubuntu-2004-022 :: ~/projects/srm-rh-ui ‹1051-rh-ui-re-work-first-steps*› » make update_vulture_whitelist                       
pipenv run vulture . --make-whitelist > whitelist.py
make: *** [Makefile:26: update_vulture_whitelist] Error 1
```

Reply:
It seems that the vulture command for some reason returns an error code 1 (even though it works exactly as expected) and then the Makefile picks it up as an error. As a workaround we've agreed to just add `|| true` at the end of the command.